compiler/src/test/java/com/squareup/anvil/compiler/dagger/ProvidesMethodFactoryGeneratorTest.kt

```diff
@@ -2276,12 +2279,12 @@ public final class DaggerComponentInterface implements ComponentInterface {
       val staticMethods = factoryClass.declaredMethods.filter { it.isStatic }
 
       val factoryInstance = staticMethods.single { it.name == "create" }
-        .invoke(null) as Factory<Any>
+          .invoke(null) as Factory<Any>
       assertThat(factoryInstance::class.java).isEqualTo(factoryClass)
 
       val providedContributingObject = staticMethods
-        .single { it.name == "provideDate" }
-        .invoke(null)
+          .single { it.name == "provideDate" }
+          .invoke(null)
```

Comment:
The indent is off here and above. It should be 2 space. Did you select the Square code style?